bucket/pixelorama.json

```
@@ -0,0 +1,26 @@
+{
+    "version": "0.6.1",
+    "description": "Open source 2D sprite editor designed for pixel art",
```

Open source is covered by license

```suggestion
    "description": "2D sprite editor designed for pixel art",
```